### ISSUE TITLE
Add support for handling validation data in Imagenet data loader

### ISSUE THREAD

**Issue by 9autilus** [govindnh4cl/squeezenet#6]
Recommendations:
* The change has to be made in `input_imagenet.py`.
* Validation images are expected to be present in a single directory `val` with names like `ILSVRC2012_val_00018454.JPEG`.
* For the ground truth, we have have file `LOC_val_solution.csv` that contains class labels and bounding boxes for object detection.
* Use `pandas` to handle CSV files.
* There is a separate class `InputImagenetTrain` for training data loader. I think we can have a single class definition for data loader of all splits (train/val/test). Just make sure to have a separate class object for each split.
